packages/sdk/src/across/lpFeeCalculator.ts

```diff
@@ -0,0 +1,38 @@
+import assert from "assert";
+import { Provider, Block } from "@ethersproject/providers";
+import { ethers, BigNumber } from "ethers";
+import { bridgePool } from "../clients";
+import { RATE_MODELS } from "./constants";
+import { BigNumberish } from "./utils";
+import { calculateRealizedLpFeePct } from "./feeCalculator";
+import { exists, BlockFinder } from "../utils";
+
+export default class LpFeeCalculator {
+  private blockFinder: BlockFinder<Block>;
+  constructor(private provider: Provider) {
+    this.blockFinder = new BlockFinder<Block>(provider.getBlock.bind(provider));
+  }
+  async getLpFeePct(tokenAddress: string, bridgePoolAddress: string, amount: BigNumberish, timestamp?: number) {
+    const rateModel = RATE_MODELS[ethers.utils.getAddress(tokenAddress)];
```

Comment:
FYI we'll soon want to read rate models from an async method `getRateModel(string tokenAddress)` which will query a `RateModelStore` contract for a specific token address